pkg/placement/membership.go

```diff
@@ -310,19 +312,32 @@ func (p *Service) disseminateOperation(targets []placementGRPCStream, operation
 
 	var err error
 	for _, s := range targets {
-		err = s.Send(o)
-
-		if err != nil {
-			remoteAddr := "n/a"
-			if peer, ok := peer.FromContext(s.Context()); ok {
-				remoteAddr = peer.Addr.String()
-			}
+		// TODO: Retry
```

Comment:
Why the TODO?

Reply:
Forgot to do a comment sweep, that was just a placeholder.